applications/audio/broadcast_configuration_tool/prj.conf

```diff
@@ -128,6 +128,8 @@ CONFIG_BT_ISO_MAX_CHAN=4
 CONFIG_BT_ISO_TX_BUF_COUNT=4
 
 CONFIG_BT_BAP_BROADCAST_SOURCE=y
+CONFIG_BT_BAP_BROADCAST_SRC_COUNT=2
```

Comment:
In general: Suggest we boost the kconfigs here so "all" combinations work out of the box.

Reply:
This is the number of BIGs. I highly doubt you will do more than 2 BIGs on air, as each will have to have a set of adv as well as the ISO data, so the air time is very quikcly cramped. We could however boost the number of BISes and subgroups